client.go

```diff
@@ -76,3 +87,39 @@ func (c *Client) From(table string) *postgrest.QueryBuilder {
 func (c *Client) Rpc(name, count string, rpcBody interface{}) string {
 	return c.rest.Rpc(name, count, rpcBody)
 }
+
+func (c *Client) SignInWithEmailPassword(email, password string) error {
```

Comment:
We will merge this first, but it will be removed soon, because it should belong to auth.